### PR TITLE
feat: add profile input to quo2

### LINKED ISSUE #15020: Implement quo2 component - Inputs/ Profile Input
Designs:
https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?node-id=15647%3A187526&t=29WaMjPTYMtHqa1n-0

Can upload photo and see preview picture
Can change text
can use emojis and special characters

there is a black list of names to use. Sync with design team to get this list.

Component tests
Preview screens

### PR DESCRIPTION
**Pull request by J-Son89**
fixes: https://github.com/status-im/status-mobile/issues/15020

Note there is only a dark blur implementation for this component designed in the Figma files.

Screenshot 2023-03-13 at 10 47 29

### CODE REVIEW

**Comment by flexsurfer** [comment]
@J-Son89 could you pls elaborate why only dark ? or you mean by design?

File: src/quo2/components/inputs/profile_input/view.cljs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+(defn profile-input
+  [{:keys [customization-color
+           placeholder
+           on-press
+           title-input-props
+           image-picker-props]}]
+  (let [full-name (:full-name image-picker-props)]
+    [rn/view
+     {:style (style/container customization-color)}
+     [rn/view {:position :relative}
```
Comment (flexsurfer) [inline src/quo2/components/inputs/profile_input/view.cljs]: `{:position :relative}` it's by default, so no need to specify

**Review by flexsurfer — APPROVED** [review]

**Comment by J-Son89** [comment]
> @J-Son89 could you pls elaborate why only dark ? or you mean by design?

apologies, should have made that clearer. Yes the Figma designs only have dark blur option

File: src/quo2/components/inputs/profile_input/view.cljs (hunk 1 above)
Comment (J-Son89) [reply src/quo2/components/inputs/profile_input/view.cljs]: thanks!

File: src/quo2/components/inputs/profile_input/view.cljs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+       {:style {}
+        :holes [{:x            33
+                 :y            23
+                 :width        24
+                 :height       24
+                 :borderRadius 12}]}
+       [user-avatar/user-avatar
+        (merge image-picker-props
+               {:customization-color customization-color
+                :full-name           (if (zero? (count full-name))
```
Comment (ilmotta) [inline src/quo2/components/inputs/profile_input/view.cljs]: Strings are also *sequences* in Clojure, so we don't need to check if the number of elements is zero. Also we have the more solid `clojure.string/blank?` that takes whitespace into account, but I don't know if it'd add much value here.

```clojure
(let [full-name ""]
  (if (seq full-name)
    full-name
    "I'm a placeholder")) ;; => I'm a placeholder

(let [full-name "someone"]
  (if (seq full-name)
    full-name
    "placeholder")) ;; => someone
```

**Review by ilmotta — APPROVED** [review]

**Pull request merged**